cf-testd/cf-testd.c

```diff
@@ -64,10 +66,12 @@ typedef struct {
 
 static const struct option OPTIONS[] =
 {
+    {"address",   required_argument, 0, 'a'},
     {"debug",     no_argument,       0, 'd'},
     {"file",      required_argument, 0, 'f'},
     {"help",      no_argument,       0, 'h'},
     {"inform",    no_argument,       0, 'I'},
+    {"port",      required_argument, 0, 'p'},
     {"timestamp", no_argument,       0, 'l'},
     {"verbose",   no_argument,       0, 'v'},
```

Comment:
why are address and port required? They weren't before.

Reply:
The options are not required. They just require arguments when used. Or am I missing something?

Reply:
@vpodzime right. looks good.